README.md

```diff
@@ -1,8 +1,12 @@
 # Northcoders News API
+1. remember to create your .env files for setting variable environments, respectively for dev and test data. These are ignored when uploaded and pulled from Github. See the setup.sql for the db names. 'npm install' will also be useful, alternatively 'npm i dotenv'. NOTE - I have set the development variable to look for '.env.dev'.
```

Comment:
Nice, this is a good start, but I would personally like a more detail explanation of exactly how to setup the .env files and what each should contain. Remember the clearer this is the easier it is to understand and follow.

You also dont need the `npm i dotenv` part as this should already be a dependency listen in your package.json file so it will be installed by just running `npm install`